README.adoc

```diff
@@ -209,16 +209,47 @@ The first is a cost vs time tradeoff. SPV proofs are expensive, so by dividing
 the cost across all of the deposits in the period, we see massive gas savings.
 This is the same model as individuals driving their own car to work vs waiting
 on the train. The second is a security measure. We need to disable the refund,
-otherwise users could get a balance and then refund their bitcoin and have
+otherwise users could get a Bank balance and then refund their bitcoin and have
 both.
 
 The entry point is `Bridge.submitDepositSweepProof` which performs the SPV
 proof, updates the wallet with the new UTXO (from
 `DepositSweep.resolveDepositSweepingWallet`), takes a deposit fee for the
-treasury (5 BPS; governable), and updates the user balances with information
+treasury (5 BPS; governable), and updates the user Bank balances with information
 from `DepositSweep.processDepositSweepTxInputs` and
 `DepositSweep.depositSweepTxFeeDistribution`.
 
+==== Redeeming
+
+An account with a Bank balance can request a redemption via
+`Bridge.requestRedemption`. We verify that the destination is valid (P2PKH,
+P2WPKH, P2SH or P2WSH), and build a redemption key based on the wallet's PKH
+and destination. There can only be one pending redemption per PKH-destination
+pair. The treasury takes a cut (`Bridge.redemptionTreasuryFeeDivisor`), and
+then we reduce the account's Bank balance and start a timer.
+
+[TIP]
+`Bridge.requestRedemption` requires a Bank balance approval to the Bridge. This
+can either be made in a separate transaction first via
+`Bank.approveBalance(Bridge.address, ...)` or in a single transaction via
+`Bank.approveBalanceAndCall(Bridge.address, ...)`.
+
+After `Bridge.redemptionTimeout`, anyone may call
```

Comment:
```suggestion
After the redemption timeout, anyone may call
```

`Bridge.redemptionTimeout` is not a thing. It's [exposed via `Bridge.redemptionParameters()`](https://github.com/keep-network/tbtc-v2/blob/938bf76a89f8b8bfdeb25ef8e322476a7919f2e7/solidity/contracts/bridge/Bridge.sol#L1610).

Reply:
We could also clarify it a bit:

```suggestion
If the redemption was not performed by the wallet, after the redemption timeout, anyone may call
```

Reply:
Gotcha! I've been using the values at https://github.com/keep-network/tbtc-v2/blob/main/solidity/contracts/bridge/Bridge.sol#L263-L292 willy nilly, but I can adjust this for sure. Do I need to also take it out of the params section?

Reply:
No, having it in parameters sections makes sense. I think we just should not refer to it via `Bridge.`. All other similar references point to functions that can actually be called.